test/functional/invalid_requests.js

```diff
@@ -1,60 +1,57 @@
-"use strict";
-
-// mocha defines to avoid JSHint breakage
-/* global describe, context, it, before, beforeEach, after, afterEach */
+'use strict';
 
 var router = module.parent.router;
 var utils = require('../utils/test_utils.js');
 var deepEqual = utils.deepEqual;
 var P = require('bluebird');
 
-describe('Invalid request handling', function() {
-    before(function () { return router.setup(); });
+describe('Invalid request handling', function () {
+    before(function () { return router.setup(); }); // eslint-disable-line max-statements-per-line
```

Comment:
if we convert it to arrow func we wouldn't need to disable eslint

Reply:
Changed to arrow function